### PR TITLE
phantomjs: homepage fixed

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by AlekSi** [comment]
:+1:

File: Library/Formula/phantomjs.rb [hunk 1]
```diff
@@ -24,7 +23,7 @@ class Phantomjs < Formula
     sha1 "887a96e55f67a3d350bc40f910926286c6cea240" => :mountain_lion
   end
 
-  head "https://github.com/ariya/phantomjs.git"
+  depends_on "openssl"
```
Comment (mikemcquaid) [inline Library/Formula/phantomjs.rb]: You can just remove this; sadly it won't use it.

File: Library/Formula/phantomjs.rb [hunk 2]
```diff
@@ -24,7 +23,7 @@ class Phantomjs < Formula
     sha1 "887a96e55f67a3d350bc40f910926286c6cea240" => :mountain_lion
   end
 
-  head "https://github.com/ariya/phantomjs.git"
```
Comment (mikemcquaid) [inline Library/Formula/phantomjs.rb]: Is this broken or something?

Comment (mikemcquaid) [inline Library/Formula/phantomjs.rb]: Ah, you moved it above, never mind.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Pull request closed without merging**